src/main/java/utils/DbConnector.java

```diff
@@ -0,0 +1,26 @@
+package utils;
+
+import org.apache.log4j.Logger;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+
+public class DbConnector {
+
+    private static final Logger dbConnectLogger = Logger.getLogger(DbConnector.class);
+    private static final String dbUrl = "jdbc:mysql://localhost:3306/shop" +
```

Comment:
```suggestion
    private static final String DB_URL = "jdbc:mysql://localhost:3306/shop" +
```
the same with login and password